.github/workflows/build.yml

```diff
@@ -57,20 +57,18 @@ jobs:
     - run: yarn run ci
     - run: yarn run lint
     - name: Get Version Number
-      uses: jozsefsallai/node-package-version@v1.0.4
-      with:
-        path: 'package.json'
-        follow-symlinks: false
+      id: getPackageInfo
+      uses: jaywcjlove/github-action-package@main
```

Comment:
Is it possible to link to a specific version of this action? I imagine this might cause issues if they ever decide to do a breaking change

Reply:
If you look at it's last major version bump it's only updating nodeJS
So while it's possible breaking changes would be introduced, I bet on he is only updating nodeJS version next major version bump
https://github.com/jaywcjlove/github-action-package/releases/tag/v2.0.0
 (I see most actions don't get updated than updated too much, i.e. include many breaking changes

![image](https://github.com/FreeTubeApp/FreeTube/assets/1018543/39cd36e5-fc71-4706-ba55-8e6b6dfc1642)